### PR TITLE
docs(nextjs): Add troubleshooting for OOM

### PR DESCRIPTION
**Pull request by chargome**
ref https://github.com/getsentry/sentry-javascript/issues/10468

Adds a troubleshooting section to nextjs for the OOM issue while building.

### CODE REVIEW

File: docs/platforms/javascript/common/troubleshooting/index.mdx [hunk 1]
```diff
@@ -537,6 +540,25 @@ Learn more about fixing these caching issues in the <PlatformLink to="/sourcemap
...
+
+  </Expandable>
+</PlatformSection>
+
+<PlatformSection supported={['javascript.nextjs']}>
+  <Expandable permalink title="Out of Memory (OOM) errors during build">
+    The problem is related to memory consumption during the build process, especially when generating source maps. Here are some potential solutions and workarounds:
+    
+    - Update your `@sentry/nextjs` package to the latest version.
+    - Increase Node.js memory limit: You can try increasing the memory limit for Node.js during the build process. Add this to your build command: `NODE_OPTIONS="--max-old-space-size=4096" next build`
```
Comment (lforst) [inline docs/platforms/javascript/common/troubleshooting/index.mdx:552]:
```suggestion
    - Increase Node.js memory limit: You can try increasing the memory limit for Node.js during the build process. Add this to your build command: `NODE_OPTIONS="--max-old-space-size=8192" next build`. This flag will increase the memory available to the node process to 8 GB. We have found that Next.js consumes around 4 GB in most cases. Decrease the size depending on your memory availability.
```

File: docs/platforms/javascript/common/troubleshooting/index.mdx [hunk 2]
```diff
@@ -450,10 +449,18 @@ As a workaround, create or modify `.npmrc` in your project root:
 ```npmrc
 shamefully-hoist=true
 ```
+
 <Alert level="warning">
   **Note**: While `shamefully-hoist=true` usually isn't the ideal solution from a dependency management perspective, it's sometimes necessary for compatibility with certain packages that expect Node.js module resolution behavior similar to npm or yarn.
 </Alert>
 
+{" "}
+<Alert level="warning">
```
Comment (coolguyzone) [inline docs/platforms/javascript/common/troubleshooting/index.mdx:458]: Looks like this note is getting repeated 👯
Screenshot 2024-12-11 at 4 21 00 PM

File: docs/platforms/javascript/common/troubleshooting/index.mdx [hunk 3]
```diff
@@ -537,6 +544,25 @@ shamefully-hoist=true
       }
     }
     ```
+
+  </Expandable>
+</PlatformSection>
+
+<PlatformSection supported={['javascript.nextjs']}>
+  <Expandable permalink title="Out of Memory (OOM) errors during build">
+    The problem is related to memory consumption during the build process, especially when generating source maps. Here are some potential solutions and workarounds:
```
Comment (coolguyzone) [inline docs/platforms/javascript/common/troubleshooting/index.mdx:553]:
```suggestion
    The problem here is related to memory consumption during the build process, especially when generating source maps. Here are some potential solutions and workarounds:
```

**Review by coolguyzone — CHANGES_REQUESTED** [review]
We need to get rid of the duplicate note, then this will be good to go!

**Pull request merged**